ulib/experimental/Steel.Memory.fst

```diff
@@ -27,6 +29,12 @@ type cell =
           perm:permission{allows_read perm} ->
           v:a ->
           cell
+  | Array: a:Type u#0 ->
+           len: nat ->
+           seq:Seq.lseq (a & perm:permission{allows_read perm}) len  ->
```

Comment:
I think the refinement on perm should be relaxed to allow some cells to have 0 permission

Reply:
See comments on disjoint below

Reply:
Indeed, changed.